pushy/src/main/java/com/eatthepath/pushy/apns/PushType.java

```diff
@@ -157,7 +157,19 @@ public enum PushType {
      * @see <a href="https://developer.apple.com/documentation/activitykit/update-and-end-your-live-activity-with-remote-push-notifications">
      *     Updating and ending your Live Activity with remote push notifications</a>
      */
-    LIVE_ACTIVITY("liveactivity");
+    LIVE_ACTIVITY("liveactivity"),
+        /**
+     * Indicates that a push notification is intended to update a running push to talk. Note that push to talk
+     * must be sent to a specific topic ({@code [base bundle ID].push-type.pushtotalk}) and include specific payload
+     * keys (see {@link ApnsPayloadBuilder#setContentState(Map)}, {@link ApnsPayloadBuilder#setEvent(LiveActivityEvent)},
+     * {@link ApnsPayloadBuilder#setDismissalDate(Instant)}, and {@link ApnsPayloadBuilder#setTimestamp(Instant)}).
```

Comment:
I understand that English isn't everybody's first language, and I appreciate the effort to include some Javadoc for this method. With that in mind and with great respect for the effort, this is not accurate. This appears to be copy/pasted from the "live activity" docs above and is not relevant for push-to-talk notifications. Reading the [push-to-talk documentation](https://developer.apple.com/documentation/pushtotalk/creating_a_push_to_talk_app), it sounds like there _are_ a number of special considerations callers will need to think about when putting together push notifications (the lifetime of device tokens, for example, and the dedicated topic), and I think it's important that we capture those here.

That said: how would you like to proceed? Would you like to try to write that on your own, or would you prefer that I provide the text?

Reply:
I'd like prefer that your provide the text